### PR TITLE
History View changes

### PR DESCRIPTION
**Pull request by ghraka**

### CODE REVIEW

**Comment by ghraka** [comment]
@giridhar : Can you please review and Merge!

File: app/controllers/news_logs_controller.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def get_document
   end
 
   def history
-     @histories = @news_log.history_tracks.order(:version => "DESC").page params[:page]
+     @histories = @news_log.history_tracks.order(:version => "ASC").page params[:page]
```
Comment (giridhar) [inline app/controllers/news_logs_controller.rb]: we wanted to have the latest change first.

Comment (ghraka) [reply app/controllers/news_logs_controller.rb]: Okay I misunderstood the requirement. I will fix it the next commit.

**Comment by ghraka** [comment]
@giridhar: Merge please!

**Pull request merged**